docs/data-sources/project.md

```diff
@@ -38,7 +38,9 @@ data "gitlab_project" "example" {
 - **http_url_to_repo** (String) URL that can be provided to `git clone` to clone the
 - **issues_enabled** (Boolean) Enable issue tracking for the project.
 - **lfs_enabled** (Boolean) Enable LFS for the project.
+- **merge_pipelines_enabled** (Boolean) Whether merged results pipelines are enabled or not.
```

Comment:
Let's use the descriptions from https://docs.gitlab.com/ee/api/projects.html

Reply:
I had that originally, it just read funny to me on a data source since it's not modifying. I'll update.

Reply:
Updated 727856f